daml-lf/archive/da/daml_lf_1.proto

```diff
@@ -1016,18 +1042,24 @@ message DefTemplate {
 // Data type definition
 message DefDataType {
   message Fields {
-    repeated FieldWithType fields = 1; // *must be non-empty*
+    repeated FieldWithType fields = 1;
```

Comment:
Did you delete the `*must be non-empty*` comment for a reason? Is this actually untrue?

Reply:
`// *must be non-empty*` was wrong since `FieldWithType` is used fro record **and variant**